web/utils/config-constants.tsx

```diff
@@ -54,15 +54,22 @@ const TEXTFIELD_TYPE_URL = 'url';
 
 export async function postConfigUpdateToAPI(args: ApiPostArgs) {
   const { apiPath, data, onSuccess, onError } = args;
-  const result = await fetchData(`${SERVER_CONFIG_UPDATE_URL}${apiPath}`, {
-    data,
-    method: 'POST',
-    auth: true,
-  });
-  if (result.success && onSuccess) {
-    onSuccess(result.message);
-  } else if (onError) {
-    onError(result.message);
+  try {
+    const result = await fetchData(`${SERVER_CONFIG_UPDATE_URL}${apiPath}`, {
+      data,
+      method: 'POST',
+      auth: true,
+    });
+    if (result.success && onSuccess) {
+      onSuccess(result.message);
+    } else if (onError) {
```

Comment:
One small thing, can `response.ok` be truthy (in `fetchData`, which would cause no exception thrown) AND the `result.success` be falsey? Just depends on your server and how it responds to different scenarios. If so, then I can leave the else if in the original code and just add the catch so we don't break anything existing that could fall under that scenario

Reply:
My guess is it's probably unlikely, but I also don't see much of a downside to catching that, just in case. Thanks for asking!